src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -602,22 +618,39 @@ bool reportNullLiteralAssignmentIfNecessary(BoundExpression expr)
                 }
                 return true;
             }
+        }
 
-            bool isDefaultOfUnconstrainedTypeParameter(BoundExpression expr)
+        private static bool IsDefaultOfUnconstrainedTypeParameter(BoundExpression expr)
```

Comment:
>IsDefaultOfUnconstrainedTypeParameter [](start = 28, length = 37)

It looks like this function is called only from the original call site. Why make it a member vs. a local function?

Reply:
To allow grouping the two `IsDefault...` methods so the similarity is more obvious.

---
In reply to: [246948367](https://github.com/dotnet/roslyn/pull/32304#discussion_r246948367) [](ancestors = 246948367)